multicluster/cmd/multicluster-controller/member_test.go

```diff
@@ -0,0 +1,123 @@
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/config/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	k8smcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+
+	mcsv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/multicluster/test/mocks"
+)
+
+var shutdownSignals = []os.Signal{os.Interrupt, syscall.SIGTERM}
+
+// The origin signal handler function will close a channel twice. If the runLeader and runMember run at the same time,
+// there will be a panic. So we need another signal handler function to avoid it.
+func mockSetupSignalHandler() context.Context {
+	ctx, cancel := context.WithCancel(context.Background())
+
+	c := make(chan os.Signal, 2)
+	signal.Notify(c, shutdownSignals...)
+	go func() {
+		<-c
+		cancel()
+		<-c
+		os.Exit(1)
+	}()
+
+	return ctx
+}
+
+var mockMemberManager *mocks.MockManager
+
+func initMemberMockManager(mockCtrl *gomock.Controller) {
+	newScheme := runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(newScheme))
+	utilruntime.Must(k8smcsv1alpha1.AddToScheme(newScheme))
+	utilruntime.Must(mcsv1alpha1.AddToScheme(newScheme))
+	fakeClient := fake.NewClientBuilder().WithScheme(scheme).WithObjects().Build()
+
+	mockMemberManager = mocks.NewMockManager(mockCtrl)
+	mockMemberManager.EXPECT().GetWebhookServer().Return(&webhook.Server{}).AnyTimes()
+	mockMemberManager.EXPECT().GetClient().Return(fakeClient).AnyTimes()
+	mockMemberManager.EXPECT().GetScheme().Return(newScheme).AnyTimes()
+	mockMemberManager.EXPECT().GetControllerOptions().Return(v1alpha1.ControllerConfigurationSpec{}).AnyTimes()
+	mockMemberManager.EXPECT().GetLogger().Return(klog.NewKlogr()).AnyTimes()
+	mockMemberManager.EXPECT().SetFields(gomock.Any()).Return(nil).AnyTimes()
+	mockMemberManager.EXPECT().Add(gomock.Any()).Return(nil).AnyTimes()
+	mockLeaderManager.EXPECT().GetConfig().Return(&rest.Config{}).AnyTimes()
+	mockMemberManager.EXPECT().Start(gomock.Any()).Return(nil).AnyTimes()
+}
+
+func TestCommands(t *testing.T) {
+	rootCommand := newControllerCommand()
+	rootCommand.AddCommand(newLeaderCommand())
+	rootCommand.AddCommand(newMemberCommand())
+
+	assert.Equal(t, "antrea-mc-controller", rootCommand.Use)
+	assert.Equal(t, "leader", newLeaderCommand().Use)
+	assert.Equal(t, "member", newMemberCommand().Use)
+}
+
+func TestRunMember(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	initMemberMockManager(mockCtrl)
+
+	testCases := []struct {
+		name        string
+		setupFunc   func(o *Options) (ctrl.Manager, error)
+		expectedRes error
+	}{
+		{
+			name: "Start member controller successfully",
+			setupFunc: func(o *Options) (ctrl.Manager, error) {
+				return mockMemberManager, nil
+			},
+		},
+	}
+
+	for _, tt := range testCases {
+		t.Run(tt.name,
+			func(t *testing.T) {
+				if tt.setupFunc != nil {
+					setupManagerAndCertControllerFunc = tt.setupFunc
+				}
```

Comment:
ditto, there is only one case, you may test it without `testCases`.

Reply:
done.